src/Core/FlatpakBackend.vala

```diff
@@ -1130,6 +1130,14 @@ public class AppCenterCore.FlatpakBackend : Backend, Object {
             return false;
         }
 
+        try {
+            foreach (var bundle_id in ids) {
```

Comment:
`foreach (unowned string bundle_id in ids) {` is required to avoid copying the string ;)